resources/ansible/roles/node/tasks/main.yml

```diff
@@ -11,27 +11,49 @@
     node_rpc_ip: "{{ ansible_host }}"
   when: provider == "digital-ocean"
 
-- name: get the latest version or a custom build of the node
+- name: download the node manager binary
   ansible.builtin.get_url:
-    url: "{{ node_archive_url }}"
-    dest: /tmp/{{ node_archive_filename }}
+    url: "{{ node_manager_archive_url }}"
+    dest: /tmp/{{ node_manager_archive_filename }}
 
-- name: extract the node binary to /usr/local/bin
+- name: extract the node manager binary to /usr/local/bin
   become: True
   ansible.builtin.unarchive:
-    src: "/tmp/{{ node_archive_filename }}"
-    dest: "{{ node_archive_dest_path }}"
+    src: "/tmp/{{ node_manager_archive_filename }}"
+    dest: "{{ node_manager_archive_dest_path }}"
     remote_src: True
 
-- name: start the node services
-  include_tasks: start_service.yml
+- name: add genesis node service
+  ansible.builtin.command:
+    argv: "{{ command_args | reject('equalto', omit) | list }}"
   vars:
-    node_rpc_port: "{{ 12000 + item }}"
-  loop: "{{ range(1, (node_instance_count | int) + 1)|list }}"
-  when: not is_genesis
+    command_args:
+      - "{{ node_manager_archive_dest_path }}/safenode-manager"
+      - add
+      - --first
+      - --port=12000
```

Comment:
You can reference the ports from the variables in `defaults`, so it would be `--port {{ node_port }}`.

Maybe a better name might actually be `genesis_port` and `genesis_rpc_port`? (I know that's not related to your PR here, but we could tidy it up now.)